### ISSUE TITLE
additional consideration for minimal technical metadata is an "Is Public" field

### ISSUE THREAD

**Issue by magelm** [NFDI4Microbiota/MetadataStandards#26]
One additional consideration for minimal technical metadata is an "Is Public" field. Sources: JGI IMG/M &  PLaBA-db (requirement unknown). NME 30May2022.

_Originally posted by @nernster in https://github.com/NFDI4Microbiota/MetadataStandards/issues/1#issuecomment-1140922652_

**Comment by MartinBoleSlo** [comment]
We can add, options along the lines of : "Data available upon reasonable request", "Public", "Available upon request", "Ethics committee approval required" ?

**Comment by magelm** [comment]
I agree that we need more options. Maybe we can fall back on existing ontologies to coin terms?
E.g. the EGA/ GA4GH created a Data Use Ontology [DUO_0000017](http://purl.obolibrary.org/obo/DUO_0000017).
DUO is an ontology which represent data use conditions. I think it was build for human-associated data, so we would need to double-check if the ontology terms fit our expectations, but the granularity of terms/ concepts is a plus IMO.
